simulator/esx/performance_manager_data.go

```diff
@@ -0,0 +1,1198 @@
+package esx
+
+var MetricData = map[string]map[int32][]int64{
+	"VirtualMachine": VmMetricData,
+	"HostSystem":     HostMetricData,
+	"ResourcePool":   ResourcePoolMetricData,
+}
+
+var VmMetricData = map[int32][]int64{
```

Comment:
Can you add some doc on how this data was generated/captured?

Reply:
See comment for vpx version!